peripherals/rcc/rcc_g4.yaml

```diff
@@ -0,0 +1,381 @@
+      Div124: [124, "pll_n_ck = vco_ck / 124"]
+      Div125: [125, "pll_n_ck = vco_ck / 125"]
+      Div126: [126, "pll_n_ck = vco_ck / 126"]
+      Div127: [127, "pll_n_ck = vco_ck / 127"]
+    PLLM: 
+      Div1: [0, "pll_p_ck = vco_ck / 1"]
+      Div2: [1, "pll_p_ck = vco_ck / 2"]
+      Div3: [2, "pll_p_ck = vco_ck / 3"]
+      Div4: [3, "pll_p_ck = vco_ck / 4"]
+      Div5: [4, "pll_p_ck = vco_ck / 5"]
+      Div6: [5, "pll_p_ck = vco_ck / 6"]
+      Div7: [6, "pll_p_ck = vco_ck / 7"]
+      Div8: [7, "pll_p_ck = vco_ck / 8"]
+      Div9: [8, "pll_p_ck = vco_ck / 9"]
+      Div10: [9, "pll_p_ck = vco_ck / 10"]
+      Div11: [10, "pll_p_ck = vco_ck / 11"]
+      Div12: [11, "pll_p_ck = vco_ck / 12"]
+      Div13: [12, "pll_p_ck = vco_ck / 13"]
+      Div14: [13, "pll_p_ck = vco_ck / 14"]
+      Div15: [14, "pll_p_ck = vco_ck / 15"]
+      Div16: [15, "pll_p_ck = vco_ck / 16"]
+    PLLSRC:
+      None: [0, "No clock sent to PLL"]
+      HSI16: [2, "No clock sent to PLL"]
+      HSE: [3, "No clock sent to PLL"]
+  
+  BDCR:
+    LSCOSEL:
+      LSI: [0, LSI clock selected"]
+      LSE: [1, "LSE clock selected"]
+    LSCOEN:
+      Disabled: [0, "LSCO disabled"]
+      Enabled: [1, "LSCO enabled"]
+    BDRST:
+      Disabled: [0, "Reset not activated"]
+      Enabled: [1, "Reset the entire RTC domain"]
+    RTCEN:
+      Disabled: [0, "RTC clock disabled"]
+      Enabled: [1, "RTC clock enabled"]
+    LSERDY:
+      _read:
+        NotReady: [0, "LSE clock not ready"]
+        Ready: [1, "LSE clock ready"]
+    RTCSEL:
+      NoClock: [0, "No clock"]
+      LSE: [1, "LSE oscillator clock used as RTC clock"]
+      LSI: [2, "LSI oscillator clock used as RTC clock"]
+      HSE: [3, "HSE oscillator clock divided by a prescaler used as RTC clock"]
+    LSEON:
+      Disabled: [0, "LSE only enabled when requested by a peripheral or system function"]
+      Enabled: [1, "LSE enabled always generated by RCC"]
+    LSECSSD:
+      _read:
+        NoFailure: [0, "No failure detected on LSE (32 kHz oscillator)"]
+        Failure: [1, "Failure detected on LSE (32 kHz oscillator)"]
+    LSECSSON:
+      "Off": [0, "CSS on LSE (32 kHz external oscillator) OFF"]
+      "On": [1, "CSS on LSE (32 kHz external oscillator) ON"]
+    LSEDRV:
+      Lower: [0, "'Xtal mode' lower driving capability"]
+      MediumLow: [1, "'Xtal mode' medium low driving capability"]
+      MediumHigh: [2, "'Xtal mode' medium high driving capability"]
+      Higher: [3, "'Xtal mode' higher driving capability"]
+    LSEBYP:
+      NotBypassed: [0, "LSE crystal oscillator not bypassed"]
+      Bypassed: [1, "LSE crystal oscillator bypassed with external clock"]
+    LSERDY:
+      _read:
+        NotReady: [0, "LSE oscillator not ready"]
+        Ready: [1, "LSE oscillator ready"]
+    LSEON:
+      "Off": [0, "LSE oscillator Off"]
+      "On": [1, "LSE oscillator On"]
+
+  CSR:
+    _modify:
+      WDGRSTF:
+        name: IWDGRSTF
+      PADRSTF:
+        name: PINRSTF
```

Comment:
These changes really belong in the `devices/common_patches/g4_rcc.yaml` file instead.

Reply:
Moved!